tools/Linux/kodi.sh.in

```diff
@@ -171,6 +171,9 @@ if command_exists gdb; then
   fi
 fi
 
+# Unset CTYPE, LANG and ALL - see issue #19883
+unset LC_CTYPE LC_ALL LANG
+
```

Comment:
do we need to unset those in further files - example https://github.com/xbmc/xbmc/blob/master/tools/Linux/kodi-standalone.sh.in ? and what are consequences of unsetting as LC_ALL is saved and restored here https://github.com/xbmc/xbmc/blob/8d098aee94781c9fad5c383f69242a69984bb69a/xbmc/windowing/X11/WinEventsX11.cpp#L175-L204

just an informational question, i'm not really deep into these things